flink-python/pyflink/table/tests/test_table_environment_api.py

```diff
@@ -440,6 +450,12 @@ def test_table_environment_with_blink_planner(self):
 
         self.assert_equals(results, ['2,hi,hello\n', '3,hello,hello\n'])
 
+    def test_set_sys_executable_for_local_mode(self):
```

Comment:
We should improve the test to eliminate the duplication. It is the same case for the other test cases in this file and there are many code duplication. You could refer to test_udf.py on how to eliminate the duplication.